src/notebook/kernel.ts

```diff
@@ -204,6 +204,16 @@ export class Kernel implements IKernel {
         // name is not needed
       } as unknown as UpdateCellIdsRequest);
     }
+
+    // Update markdown cells to run on change.
+    const markdownChanges = e.cellChanges.flatMap((change)=>change.cell).filter((cell) =>
```

Comment:
I think this introduces a bug where the notebook will keep executing if somehow the "markdown" cell fails.
(e.g. regex missing quoting, or `mo` is not imported)